daphne_worker_test/tests/test_runner.rs

```diff
@@ -89,8 +89,8 @@ impl TestRunner {
             DapVersion::Draft04 => "v04",
             _ => panic!("unimplemented DapVersion"),
         };
-        let mut leader_url = Url::parse(&format!("http://leader:8787/{}/", version_path)).unwrap();
-        let mut helper_url = Url::parse(&format!("http://helper:8788/{}/", version_path)).unwrap();
+        let mut leader_url = Url::parse(&format!("http://leader:8080/{}/", version_path)).unwrap();
+        let mut helper_url = Url::parse(&format!("http://helper:8080/{}/", version_path)).unwrap();
```

Comment:
In order to conveniently run tests locally, it's helpful to have the Leader and Helper on separate ports so that they can use the same address (localhost).